content/departments/product-engineering/process/planning-process.md

```diff
@@ -0,0 +1,105 @@
+
+Product/Engineering also has a [department level strategy](../strategy-goals/index.md) that describes where we want our product to be at the end of the year and how we are going to achieve that.
+
+These strategies are intended to be relatively stable over the course of a year, but anyone can propose edits or adjustments at any time as we learn new information.
+
+### Per-team strategy pages
+
+We also use [per-team strategy pages](../../../strategy-goals/strategy/index.md#team-strategy-pages) as a [narrative framework](https://www.mindtheproduct.com/the-importance-of-narrative/) for prioritization, to document trade-offs between goals, and to communicate strategy outwardly. Because this is a longer outlook, they tend to not change much each month but should be reviewed. Be sure to @ mention your engineering, design, product marketing, and any other important partners for feedback in your monthly update PR.
+
+Engineering should feel empowered to bring up that they feel strongly about in conversations with product. [We want to push decisions down to people closest to those problems](../../../company-info-and-process/communication/decisions.md#what-makes-an-effective-decision). It is product's responsibility to help give insight into customer pains and feedback, strategic priorities, and to ensure consistency across the product.
+
+We ask that teams leverage the [Strategy page template](https://github.com/sourcegraph/handbook/blob/main/page_templates/strategy_template.md) when creating and updating content.
+
+## Quarterly department OKRs
+
+Each quarter we set OKRs at the department level (all of product and engineering).
+
+OKRs define what success looks like each quarter, taking into account where we are at as a company and what we think the next things we need to accomplish given the vision and strategy we have for the year.
+
+We don't require OKRs at lower levels (orgs, teams) because the most important artifact is the roadmap (using strategy and department OKRs as an input). Orgs and/or teams may decide to set their OKRs if they find that helpful (but this doesn't replace having a roadmap).
+
+### OKR tracking
+
+The source of truth for current OKRs and status of those OKRs is in our [private issue tracker](https://github.com/sourcegraph/product-engineering-tracker/labels/summary).
+
+- This is private because KRs frequently contain sensitive business metrics and customer names).
+- The sanitized public version of our OKRs are [here in the handbook](../../../strategy-goals/goals/index.md).
+
+Each KR has an associated GitHub issue following [this template](https://github.com/sourcegraph/product-engineering-tracker/issues/new?template=product-engineering-kr.md).
+
+- The assignee of each OKR is responsible for posting an update to the associated tracking issue by 19:00 UTC each Friday. Weekly updates should include:
```

Comment:
I think this is the area I had a bit of confusion on...

_The assignee of each OKR is responsible for posting an update to the associated tracking issue by 19:00 UTC each Friday._ To post an update, create a new comment and include the current status along with the following details:

Reply:
```suggestion
- The assignee of each OKR is responsible for posting an update to the associated tracking issue by 19:00 UTC each Friday. To post an update, create a new comment with the following details:
```